packages/desktop-client/src/components/reports/DateRange.tsx

```diff
@@ -1,16 +1,21 @@
-import React from 'react';
+import React, { type ReactElement } from 'react';
 
 import * as d from 'date-fns';
 
 import { theme } from '../../style';
 import Block from '../common/Block';
 
-function DateRange({ start, end }) {
-  start = d.parseISO(start);
-  end = d.parseISO(end);
+type DateRangeProps = {
+  start: string | Date;
+  end: string | Date;
+};
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏the input will always be a `string`. But I see why you did this (to re-use `start` & `end` variables). Lets solve that in a different way (will comment below).
```suggestion
type DateRangeProps = {
  start: string;
  end: string;
};
```